commands/Moderation/unmute.js

```diff
@@ -0,0 +1,27 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+			description: 'Unmutes a mentioned user.',
+			usage: '<member:user> [reason:string] [...]',
```

Comment:
member arg should use the member argument as you cannot remove roles from members who are not present in the server.